rover/source/docs/rover-auxiliary-functions.rst

```diff
@@ -4,109 +4,93 @@
-For example, if you would prefer to use channel 9, set :ref:`AUX_CH <AUX_CH>` to "9".
+Any RC input channel can be assigned any auxiliary function. However the RC channel should not be used by any other 
+function. Typically channels above 7 are free to be used for auxiliary functions.
 
 Assigning the Feature
 =====================
 
-The :ref:`CH7_OPTION <CH7_OPTION>` parameter controls which feature is assigned to the auxiliary switch.  By default this is the "Save Waypoint" feature.
+The :ref:`RC7_OPTION <RC7_OPTION>` parameter controls which feature is assigned to RC input 7. Each RC channel has its 
+own RCx_OPTION parameter that can be accessed in the full parameter list.
 
-If using the Mission Planner, the Config/Tuning >> Basic Tuning's "Ch7 Opt" drop-down can be used to easily set the desired feature.
+If using the Mission Planner, the Config/Tuning >> Basic Tuning's "RC7 Opt", "RC8 Opt, "RC9 Opt" and "RC10 Opt" drop-down can be used to easily set the desired feature.
 
 .. image:: ../images/rover-auxiliary-functions-MP.png
     :target: ../_images/rover-auxiliary-functions-MP.png
 
 Supported Features
 ==================
 
-.. raw:: html
-
-   <table border="1" class="docutils">
-   <tbody>
-   <tr>
-   <th>CH7_OPTION</th>
-   <th>Feature</th>
-   <th>Description</th>
-   </tr>
-   <tr>
-   <td>0</td>
-   <td><strong>Disabled</strong></td>
-   <td>No feature assigned
-   </td>
-   </tr>
-   <tr>
-   <td>1</td>
-   <td><strong>Save Waypoint</strong></td>
-   <td>The vehicle's position will be stored as a new waypoint in the mission command list.  Use Auto mode later to return to this location
-   </td>
-   </tr>
-   <tr>
-   <td>2</td>
-   <td><strong>Learn Cruise Speed</strong></td>
-   <td>Record the vehicle's speed and throttle to the CRUISE_SPEED and CRUISE_THROTTLE parameters
-   </td>
-   </tr>
-   <tr>
-   <td>3</td>
-   <td><strong>Arm / Disarm</strong></td>
-   <td>Arm or Disarm the vehicle
-   </td>
-   </tr>
-   <tr>
-   <td>4</td>
-   <td><strong>Manual</strong></td>
-   <td>Change to Manual mode
-   </td>
-   </tr>
-   <tr>
-   <td>5</td>
-   <td><strong>Acro</strong></td>
-   <td>Change to Acro mode
-   </td>
-   </tr>
-   <tr>
-   <td>6</td>
-   <td><strong>Steering</strong></td>
-   <td>Change to Steering mode
-   </td>
-   </tr>
-   <tr>
-   <td>7</td>
-   <td><strong>Hold</strong></td>
-   <td>Change to Hold mode
-   </td>
-   </tr>
-   <tr>
-   <td>8</td>
-   <td><strong>Auto</strong></td>
-   <td>Change to Auto mode
-   </td>
-   </tr><tr>
-   <td>9</td>
-   <td><strong>RTL</strong></td>
-   <td>Change to RTL mode
-   </td>
-   </tr>
-   <tr>
-   <td>10</td>
-   <td><strong>SmartRTL</strong></td>
-   <td>Change to SmartRTL mode
-   </td>
-   </tr>
-   <tr>
-   <td>11</td>
-   <td><strong>Guided</strong></td>
-   <td>Change to Guided mode
-   </td>
-   </tr>
-   </tbody>
-   </table>
++----------------------+----------------------------+
```

Comment:
I wonder if we should just link to the parameters page so this doesn't get out of date?

Reply:
probably, I was planning to do a more in-depth explanation of each thing, but I realised I was not really sure what they all did.